### ISSUE TITLE
Request FeatureToogleProvider configurable onLoad module

### ISSUE THREAD

**Issue by humbertosales** [willmendesneto/ngx-feature-toggle#156]
```
[ ] bug report => search github for a similar issue or PR before submitting
[ x] feature request
```

The FeatureToogleProvider is one directive, but we need load map features onLoad of application. The features are valid for all aplication.
Why set list features in HTML?

Please, can you create provider for load on module and, prefer, load rxJS features?

**Comment by willmendesneto** [comment]
Hi @humbertosales . Thanks for open this issue.

There are some reason why rxjs is not used in this project:

- Add `rxjs` in this repository will increase the component bundle, which is not ideal. However you can use `rxjs` or any other alternative crearting a component wrapper and adding this value into `feature-toggle-provider`;
- There are other ways to update this data. Since `rxjs` should be a project dependency and the component accepts changes by binding after changes released on `ngx-feature-toggle@4.1.0` (more details in this issue https://github.com/willmendesneto/ngx-feature-toggle/issues/81);

Apart from that I can't understand/find any scenario that `onLoad` since the current approach covers all the reasonable scenarios. 

Could you elaborate/clarify more about your issue please? It will be great if you can share a stackblitz link with your issue based on [this template](https://stackblitz.com/edit/ngx-feature-toggle-sample) will be great to understand more about your request.

https://stackblitz.com/edit/ngx-feature-toggle-sample

**Comment by willmendesneto** [comment]
Closing this issue by lack of reply
